kaldb/src/test/java/com/slack/kaldb/server/KaldbTest.java

```diff
@@ -53,16 +53,17 @@ public class KaldbTest {
   private static final String KALDB_TEST_CLIENT_1 = "kaldb-test-client1";
   private static final String KALDB_TEST_CLIENT_2 = "kaldb-test-client2";
 
-  private static KaldbSearch.SearchResult searchUsingGrpcApi(String queryString, int port) {
+  private static KaldbSearch.SearchResult searchUsingGrpcApi(
```

Comment:
Is there a way we can check here that we are only hitting the nodes we want to hit when performing the distributed search?

Reply:
In KaldbDistributedQueryServiceTest, we are just ensuring that function works as expected given the metadata in the store. 

It would be good to also have an end to end test where some nodes are hit and some nodes are not hit. Also, would be good to add a check to make sure that we are making one search request per chunk.

Reply:
we do that in KaldbTest.java

Reply:
it's in KaldbTest.java